projectPage.css

```diff
@@ -70,7 +49,7 @@ img {
 }
 
 
-.QT {
```

Comment:
changed this to project-header for descriptiveness, feel free to change back to QT if QT is a meaningful name for you.